docs/ref/contrib/auth.txt

```diff
@@ -111,39 +111,56 @@ Fields
         A datetime designating when the account was created. Is set to the
         current date/time by default when the account is created.
 
-Methods
--------
+Attributes
+----------
 
 .. class:: models.User
 
-    .. method:: get_username()
-
-        Returns the username for the user. Since the User model can be swapped
-        out, you should use  this method instead of referencing the username
-        attribute directly.
-
-    .. method:: is_anonymous()
+    .. attribute:: is_anonymous
 
-        Always returns ``False``. This is a way of differentiating
-        :class:`~django.contrib.auth.models.User` and
+        Read-only attribute which is always ``False``. This is a way of
+        differentiating :class:`~django.contrib.auth.models.User` and
         :class:`~django.contrib.auth.models.AnonymousUser` objects.
         Generally, you should prefer using
-        :meth:`~django.contrib.auth.models.User.is_authenticated()` to this
-        method.
+        :attr:`~django.contrib.auth.models.User.is_authenticated` to this
```

Comment:
Since `is_authenticated` is the preferred variant, it should be documented before `is_anonymous`.